admin-client/src/pages/organization/Index.svelte

```diff
@@ -52,7 +66,19 @@
         </ul>
       </td>
       <td>
-        <button class="usa-button usa-button--secondary">Delete</button>
+        {#if org.isActive}
+        <button
+          class="usa-button usa-button--secondary"
+          on:click={deactivate(org.id)}>
```

Comment:
Does this work? Shouldn't it be
```suggestion
          on:click={() => deactivate(org.id)}>
```

Reply:
it's not about being idiomatic, they are different. The original code SHOULD be calling the function immediately and passing it's result as the click handler, while the second is passing the function itself. We want the latter. The first shouldn't work and makes me wonder what Svelte is doing.

Reply:
I see what you mean – and now that I see it I also wonder what Svelte is doing. I'm going to try it again to confirm that I was seeing what I thought I was seeing, record that result, and then try changing it.

Reply:
Yes, it does work as you see it, and it doesn't activate/deactivate until you click it.

Reply:
:jinx:

Reply:
Yes, with that fix (which I'm linting and testing before pushing) this one looks like it'll be OK. I'll just be a few minutes.